components/NavigationBar/src/MDCNavigationBar.h

```diff
@@ -106,6 +106,12 @@ IB_DESIGNABLE
  */
 @property (nonatomic, strong, null_resettable) UIFont *titleFont;
 
+/**
+ The Material inkColor that is used for all buttons in trailing and leading button bars. Only
+ affects buttons that are created after changing the property.
```

Comment:
Document that if this property is nil then the default button ink color will be used instead.

Reply:
Done.